src/helpers/translation.ts

```diff
@@ -1218,6 +1219,11 @@ export const memberMessages = {
       id: 'member.text.deleteMemberNoteConfirmation',
       defaultMessage: '備註一經刪除即不可恢復，確定要刪除嗎？',
     },
+    deleteMemberConfirmation: {
+      id: 'member.text.deleteMemberConfirmation',
+      defaultMessage: '會員一經刪除即不可恢復，會將該會員的相關資料與權限移除，確定要刪除嗎？'
+
+    },
```

Comment:
Style-based format is being gradually phased out.
Please gradually transition to a component-based format.
FYI: 
https://www.notion.so/kkshyu/Code-Review-c29ef181f60d489e939869373e77a065?pvs=4#f92f240916614d76851f009c95be27ce

Reply:
And it's better to write translation in English.